include/dmlc/registry.h

```diff
@@ -188,8 +188,8 @@ class FunctionRegEntryBase {
  */
 #define DMLC_REGISTRY_ENABLE(EntryType)                                 \
   template<>                                                            \
-  Registry<EntryType> *Registry<EntryType>::Get() {                     \
-    static Registry<EntryType> inst;                                    \
+  Registry<EntryType > *Registry<EntryType >::Get() {                   \
```

Comment:
EntryType could contain something that ends with ```>```, which I am not sure recognized by old c++ compiler(should be ok for c++11 compatible ones) 

```vector<vector<int>> ``` vs ```vector<vector<int> > ```